src/lang/locales.json

```diff
@@ -1,14 +1,17 @@
 [
   {
     "code": "en-US",
-    "name": "English"
+    "name": "English",
+    "slug": "en-US"
```

Comment:
I think this is probably a good way of allowing the slug to be configurable by other systems, even if it matches the language tag exactly by default. 👍